src/components/Slider/Slider.tsx

```diff
@@ -57,6 +63,91 @@ export const BoemlySlider: React.FC<SliderProps> = ({
     setInputValue(event.target.value);
   };
 
+  const renderBoundarySlider = () => (
```

Comment:
Is there a reason why you are not returning it directly?

```suggestion
  if (boundaryVariant) return (
```

Reply:
No there was no reason. Changed it now.